### ISSUE TITLE
Some questions about training

### ISSUE THREAD

**Issue by zsh1223** [md4all/md4all#24]
1、 In the evaluation, what is the difference between “eval-md4allDDa-wo-daytime-norm-80m-nuscenes-val”、“eval-md4allDDa-wo-daytime-norm-80m-nuscenes-test”、“eval_baseline_80m_nuscenes_val”、“eval_md4allDDa_80m_nuscenes_val”？
       Is the difference between“eval-md4allDDa-wo-daytime-norm-80m-nuscenes-val” and “eval-md4allDDa-wo-daytime-norm-80m-nuscenes-test” evaluated using both the val dataset and the test dataset. Which is the more authoritative or universal evaluation method?
       What is the difference between “eval_md4allDDa_80m_nuscenes_val” and “eval-md4allDDa-wo-daytime-norm-80m-nuscenes-val”? I don't quite understand the meaning of "eval md4allDDa wo daytime norm 80m popularity val"?
       Does "eval_baseline_80m_musenes_val" mean only evaluating the baseline model separately (monodepth2+weak speed supervision)?
2、I used the pre trained model you provided but couldn't obtain the results in Table 1 of the paper. May I ask if the pre trained model you provided is not the one in the paper?
<img width="415" alt="image" src="https://github.com/md4all/md4all/assets/130481211/1fa5b62e-7acf-42af-bd6f-e8a5bed4b204">
<img width="415" alt="image" src="https://github.com/md4all/md4all/assets/130481211/0775044c-b0ec-4369-b6a6-9a7ddca8224e">
<img width="416" alt="image" src="https://github.com/md4all/md4all/assets/130481211/3779afcb-7df9-48a1-a145-5f8e7318148c">
What does a1	a1_gt	a1_pp	a1_pp_gt	a2	a2_gt	a2_pp	a2_pp_gt	a3	a3_gt	a3_pp	a3_pp_gt	abs_rel	abs_rel_gt	abs_rel_pp	abs_rel_pp_gt	count	rmse	rmse_gt	rmse_log	rmse_log_gt	rmse_log_pp	rmse_log_pp_gt	rmse_pp	rmse_pp_gt	sq_rel	sq_rel_gt	sq_rel_pp	sq_rel_pp_gt mean in the evaluation indicators, and why are there expressions like "_gt", "_pp", and "pp_gt"?
3、Is my command below correct?
Train the sup=Mv, md4all-DD command: python train.py --config <PATH_TO_MD4ALL>/config/train_md4allDDa_nuscenes.yaml
Evaluate the sup=Mv, md4all-DD command: 
“python evaluation/evaluate_depth.py --config <PATH_TO_MD4ALL>/config/eval_md4allDDa_80m_nuscenes_val.yaml”
or “python evaluation/evaluate_depth.py --config <PATH_TO_MD4ALL>/config/eval-md4allDDa-wo-daytime-norm-80m-nuscenes-test”

**Comment by sgasperini** [comment]
Hi @zsh1223,

Thank you again for your interest in our work. I will answer to your questions below:

1. The *`val` evaluations are on the validation set, which is what we reported for nuScenes in the paper (e.g., Table 1). The *`test` evaluations are on the test set, which we reported for nuScenes only in the supplementary material. We would suggest you do the same, and focus on the validation set.
The term `wo-daytime-norm` indicates that such evaluation does not use the time-dependent normalization but normalizes all inputs the same way. If such a term is not present, then the normalization is done differently for day images than night ones. A more detailed explanation can be found in issue #8 and in Section A.1.3 of our supplementary material.

2. Your results look good. As written in issues #8, #11, and #18, the `gt`, `pp`, and `pp_gt` depend on the gt-median scaling (`gt`) or test-time flipping (`pp`). For the baseline and ours, you should look at the `pp` values. Your `pp` values for md4allDDa look pretty much the same as what we reported in Table 1 (in your screenshots). The small variation can be due to the different hardware or libraries you are using. For example, you seem to be using conda, while the numbers we reported were obtained with docker.
For your baseline screenshot, it looks like you did not use our baseline but the md2 model we shared instead (the scores are the same as md2 in Table 1, second line). Please consider that for md2, you have to look at the `pp_gt` values since its predictions are only up to scale. If you want to evaluate our baseline, you should use our pre-trained baseline model [from here (shared G-Drive)](https://drive.google.com/file/d/1RN_k6eXL3pHUPFOSX2j05wfbChxYFcO1/view?usp=share_link).

3. Your commands are the same as we reported in the readme of this repository, so they are ok.

I hope this clears your doubts. If you have follow-up questions, let us know.

**Comment by zsh1223** [comment]
You replied so quickly!!!!!!
Thank you for your answer

**Comment by sgasperini** [comment]
Hi! Closing this for inactivity. Feel free to reopen it if you have follow-up questions.
